object-cache.php

```diff
@@ -825,6 +825,46 @@ function ( element ) {
 		";
 	}
 
+	/**
+	 * Returns the collected raw stats.
+	 *
+	 * @return array $stats
+	 */
+	function get_stats() {
+		$stats = [];
+		$stats['totals'] = [
+			'query_time' => $this->time_total,
+			'size' => $this->size_total,
+		];
+		$stats['stats'] = $this->stats;
+		$stats['operations'] = [];
+		$stats['groups'] = [];
+		foreach ( $this->group_ops as $cache_group => $dataset ) {
+			if ( empty( $cache_group ) ) {
+				$cache_group = 'default';
+			}
+
+			if ( ! isset( $stats['groups'][ $cache_group ] ) ) {
+				$stats['groups'][ $cache_group ] = true;
```

Comment:
Do we need this part returned?

Reply:
It's just to reduce the post-processing needed on the output side

Reply:
Can we return a flat array of groups instead? 

```
$stats['groups'] = [ 'group1', 'group2' ];
```

Reply:
@WPprodigy I specifically chose this way because of the `isset()` check would be faster than `in_array()`.

Reply:
Hmm, I'm just not a big fan of doing this for small datasets + things that don't run often. Particularly when it makes for a rather odd data structure in what's intended to be the main "public" api portion.  We wouldn't expect a data structure like this to be returned from a REST API for example.  

Ultimately either way doesn't really make any notable difference, so not a blocker or anything.